src/others/components/map/Map.tsx

```diff
@@ -77,10 +87,22 @@ export const Map = ({ sourceWithLayer, interactiveLayerIds }: MapProps) => {
               totalItems: requestData.amount,
             },
           });
+
+          if (hoveredRegionId) {
+            mapRef.current.setFeatureState(
+              { source: 'state', id: hoveredRegionId },
+              { hover: false }
+            );
+          }
+          setHoveredRegionId(features[preferredLayerIndex].id);
```

Comment:
I've getting error "Error: The feature id parameter must be provided" and when I logged here "features[preferredLayerIndex]" I see that id is undefined so there is probably some more places in which newly moved/renamed id (from properties to feature object) is not propagated.